### DISCUSSION TITLE
[Feature Request] Velocity Aproximation C_VEL

### DISCUSSION THREAD

**Discussion by ghost** [visose/Robots#92] [Q&A] [answered]
Hi Vicente,

Just wondering if it'd be convient for you to add in C_VEL aproximation?
The way I'm doing it now is generate the SRC, then in notepad replace APO.CDIS and C_DIS with APO.CVEL and C_VEL, then change the zone value for APO.CVEL, but when I want to use both C_DIS and C_VEL, it become very complicated to filter out the different tool paths.

ref (p.13): http://www.wtech.com.tw/public/download/manual/kuka/krc4/KUKA%20KRL-Syntax%208.x.pdf

thanks.

**Comment by visose** [comment, marked as answer]
You can use the custom code component combined with Grasshopper's text replace component to do this, no need to edit in notepad. This way it will do this patch automatically every time the program is updated.

As a quick fix for only selecting some targets, you could create a command that is just a comment "cvel" and attach it to those targets, then you use the text replace component to replace a new line character plus that comment with C_VEL.

I won't add C_VEL anytime soon sorry.
